airframe-http-finagle/src/test/scala/wvlet/airframe/http/finagle/FinagleTest.scala

```diff
@@ -64,4 +63,40 @@ class FinagleTest extends AirSpec {
     r.contentBytes shouldBe "hello world".getBytes(StandardCharsets.UTF_8)
     resp.toRaw shouldBeTheSameInstanceAs resp
   }
+
+  test("convertToFinagleRequest test") {
+    test("basic") {
+      val req = HttpMessage.Request(HttpMethod.GET, "/foo")
+
+      val response = convertToFinagleRequest(req)
+
+      response.method.name shouldBe "GET"
+      response.path shouldBe "/foo"
+    }
+    test("GET with query parameters") {
+      val req = HttpMessage.Request(HttpMethod.GET, "/foo?bar=true&baz=1234")
+
+      val response = convertToFinagleRequest(req)
+
+      response.method.name shouldBe "GET"
+      response.path shouldBe "/foo"
```

Comment:
I thought the path should contain the query string `?bar=true&baz=1234`. Does it need to be set as params?

Reply:
In my understanding, path doesn't contain a query string.
And, according to the [wikipedia](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier), behavior of the `uri` parameter in the case class, `Request` ,  is different with the definition of URI in the wikipedia.
I was actually confused with it before.

https://github.com/wvlet/airframe/blob/8ef23a39ea5b12f17ac9415abe956275108fb225/airframe-http/src/main/scala/wvlet/airframe/http/HttpMessage.scala#L207-L214